mobilitydb_sqlalchemy/types/Period.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+
+from pymeos.temporal import TText
+from sqlalchemy.types import UserDefinedType
+
+from .TBaseType import TBaseType
+
+def period_ident(self,t):
+    return(t)
+
+class Period(TBaseType):
+    pymeos_instant_type = TText
+    pymeos_deserializer_type = period_ident
+
+    def get_col_spec(self):
+        return "PERIOD"
+
+    @staticmethod
+    def validate_type(value):
+        if not str(value):
+            raise TypeError(
+                "Period needs well-formatted string Got: {} Expected {}".format("??", "?")
+            )
+
+    def bind_processor(self, dialect):
+        def process(value):
+            self.validate_type(value)
+            return(value)
+        return process
+
+    def result_processor(self, dialect, coltype):
+        def process(value):
+            return(value)
```

Comment:
Perhaps we can return value as a `Period` object from PyMEOS. That way we would be able to operate over the returned values a bit more easily, like for example to do `period.lower` to get the lower bound.
```suggestion
            return MEOSPeriod(value)
```
Of course, to do this we would need to import it first
```python
from pymeos.time import Period as MEOSPeriod
```
Note that I am renaming the import so that it does not clash with our own `Period` class here.